packages/lesswrong/server/resolvers/ultraFeedResolver.ts

```diff
@@ -54,6 +54,8 @@ export const ultraFeedGraphQLTypeDefs = gql`
     commentMetaInfos: JSON
     comments: [Comment!]!
     post: Post
+    isOnReadPost: Boolean
+    postSources: JSON
```

Comment:
I don't like these JSON blogs; we have much better affordance to properly create graphql object types now.